internal/api/verify.go

```diff
@@ -92,27 +133,19 @@ func (a *API) verifyGet(w http.ResponseWriter, r *http.Request) error {
 		token       *AccessTokenResponse
 		authCode    string
 	)
-	var flowType models.FlowType
+	flowType := models.ImplicitFlow
 	var authenticationMethod models.AuthenticationMethod
 	if strings.HasPrefix(params.Token, PKCEPrefix) {
 		flowType = models.PKCEFlow
 		authenticationMethod, err = models.ParseAuthenticationMethod(params.Type)
 		if err != nil {
 			return err
 		}
-	} else {
-		flowType = models.ImplicitFlow
-	}
```

Comment:
@J0 i've set the initial value of `flowType` to implicit flow so we can remove these few lines